### DISCUSSION TITLE
Aprendiendo sobre GitHub

### DISCUSSION THREAD

**Discussion by agonzalezmx69** [skills/.github#198] [General]
Hoy sé algo más que ayer. Esto es un desafío. Hay que salir de la zona de confort

**Comment by marcocetos** [comment]
eso mismo ! siempre salir de la zona de comfort ! si podemos !!

**Reply by eztlicoatl91** [reply]
Todo es posible.

**Comment by Pulse-61** [comment]
I agree!!!  This is definitely challenging me, but I am rising to the challenge, Good Luck to everyone!!!  Adios!!!
